src/bugreporter/__init__.py

```diff
@@ -0,0 +1 @@
+from odemis_bugreporter import OdemisBugreporter
```

Comment:
I added it, so in other modules we can call `from bugreporter import ...` instead of `from bugreporter.odemis_bugreporter import ...` like we did for example in the stitching module. Does this make sense?

Reply:
I added it, so in other modules we can call `from bugreporter import ...` instead of `from bugreporter.odemis_bugreporter import ...` like we did for example in the stitching module. Does this make sense?

Reply:
I added it, so in other modules we can call `from bugreporter import ...` instead of `from bugreporter.odemis_bugreporter import ...` like we did for example in the stitching module. Does this make sense?

Reply:
I added it, so in other modules we can call `from bugreporter import ...` instead of `from bugreporter.odemis_bugreporter import ...` like we did for example in the stitching module. Does this make sense?

Reply:
abc